README.md

```diff
@@ -30,3 +30,4 @@ To learn how to use **function-framework-go** and how it works, refer to the [fu
 | v0.0.0-20210628081257-4137e46a99a6 | v0.3.*       | v0.1.0  | v0.2.2 (openfunction/builder-go:v0.2.2-1.15) |
 | v0.0.0-20210922063920-81a7b2951b8a | v0.4.*       | v0.2.0  | v0.3.0 (openfunction/builder-go:v0.3.0-1.15) |
 | v0.1.1                             | v0.5.*       | v0.2.0  | v0.4.0 (openfunction/builder-go:v0.4.0-1.15) |
+| v0.2.0                             | v0.6.*       | v0.3.0  | v0.5.0+                                      |
```

Comment:
The builder could be [v2-1.16](https://hub.docker.com/layers/187319070/openfunction/builder-go/v2-1.16/images/sha256-c2b1fbaea4e42ad988c2901125c853cb826e94c233b325d710ce2577b7441ee9?context=repo)